frameworks/cassandra/build.sh

```diff
@@ -9,7 +39,12 @@ ROOT_DIR="$(dirname "$(dirname $FRAMEWORK_DIR)")"
 export TOOLS_DIR=${ROOT_DIR}/tools
 BUILD_DIR=$FRAMEWORK_DIR/build/distributions
 PUBLISH_STEP=${1-none}
-${ROOT_DIR}/gradlew distZip -p ${ROOT_DIR}/sdk/executor
+
+if [ x"$cli_only" = xtrue ]; then
```

Comment:
This is strongly un-idiomatic.  It happens to work, but I've never seen this pattern in 25 years of shell scripting.  it's fragile, "True" is false.  I'm not a fan.